--- a/emsdk/fastcomp/emscripten/src/settings.js
+++ b/emsdk/fastcomp/emscripten/src/settings.js
@@ -522,7 +522,7 @@
 //     -fno-exceptions to really get rid of all exceptions code overhead,
 //     as it may contain thrown exceptions that are never caught (e.g.
 //     just using std::vector can have that). -fno-rtti may help as well.
-var DISABLE_EXCEPTION_CATCHING = 1;
+var DISABLE_EXCEPTION_CATCHING = 0;
 
 // Enables catching exception in the listed functions only, if
 // DISABLE_EXCEPTION_CATCHING = 2 is set
